controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -417,13 +407,66 @@ func (r *ROSAControlPlaneReconciler) rosaClusterToROSAControlPlane(log *logger.L
 	}
 }
 
-func (r *ROSAControlPlaneReconciler) getOcmCluster(rosaScope *scope.ROSAControlPlaneScope, ocmConnection *sdk.Connection) (*cmv1.Cluster, error) {
-	clusterKey := rosaScope.ControlPlane.Name
+// OCMClient is a temporary helper to talk to OCM API.
+// TODO(alberto): vendor this from https://github.com/openshift/rosa/tree/master/pkg/ocm or build its own package here.
```

Comment:
Is this vendorable? Why have we decided not to import it today?

Reply:
We would have to vendor the complete rosa cli repo https://github.com/openshift/rosa, I don't think it is possible to only import the ocm package